<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelConfigWidget</class>
 <widget class="QWidget" name="ChannelConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_show">
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_selColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_SystemIdentMethod">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>System Ident. Method:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_SysIdentMethod"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_SysIdentWindow"/>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_CalcDelay">
       <property name="text">
        <string>Calc Delay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Delay">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Delay (Samples):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Delay"/>
     </item>
     <item>
      <widget class="QLabel" name="label_Offset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Offset (Samples):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Offset"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SetDelayOffset">
       <property name="text">
        <string>Set Delay / Offset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_N">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>System Length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_N"/>
     </item>
     <item>
      <widget class="QLabel" name="label_L">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Identification Length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_L"/>
     </item>
     <item>
      <widget class="QLabel" name="label_Nfft">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Frequency Length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Nfft"/>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ExpTimeSmoothing">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Exp. Time Smooting:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_ExpTimeSmoothing">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_WindowType"/>
     </item>
     <item>
      <widget class="QLabel" name="label_WindowWidth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Window (Half) Width:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_WindowWidth"/>
     </item>
     <item>
      <widget class="QLabel" name="label_WindowOffset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Window Offset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_WindowOffset"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SetWindow">
       <property name="text">
        <string>Set Window Width / Offset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_SaveIR">
         <property name="text">
          <string>Save IR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_LoadIR">
         <property name="text">
          <string>Load IR</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Update">
       <property name="text">
        <string>Update View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
